routes/course.js

```diff
@@ -112,6 +112,14 @@ router.post("/api/courses", auth, async (req, res) => {
           message: "Invalid combination of plan_id and distribution_ids.",
         });
     });
+    const retrievedCourses = await Courses.findByPlanId(course.plan_id); 
```

Comment:
wow this is great!

Reply:
if you have concerns about the efficiency of this, we shall see how this add course route does in the local deployment :))

Reply:
Yea, we were a little worried that adding another api call might slow it down too much, but wasn't too sure how else to check the other classes in the plan(at least in backend). In case this turns out to be too inefficient, our alternative was trying to check the courses on the frontend instead.

Reply:
yeah let's keep that in mind! I think this will be fine though.